test-kotest5/build.gradle

```diff
@@ -41,11 +41,11 @@ dependencies {
 }
 
 test {
-    useJUnitPlatform()
-}
-
-// PTS just skips Kotest tests silently...
-tasks.withType(Test).configureEach {
+    useJUnitPlatform {
+        includeEngines("kotest")
```

Comment:
Why is this needed?

Reply:
It's not strictly necessary, but makes it explicit that only Kotest should be used by JUnit Platform. I noticed there are also [Spock](https://ge.micronaut.io/s/6fqdgpmvcgajq/dependencies?configurationFilter=WyJ0ZXN0UnVudGltZUNsYXNzcGF0aCJd&dependencies=spock&expandAll&focusedDependencyView=versions&toggled=W1swXSxbMl0sWzNdXQ) and the [JUnit Jupiter engine](https://ge.micronaut.io/s/6fqdgpmvcgajq/dependencies?configurationFilter=WyJ0ZXN0UnVudGltZUNsYXNzcGF0aCJd&dependencies=jupiter-engine&expandAll&focusedDependencyView=versions&toggled=W1swXSxbMl1d) on the test runtime classpath (see linked Build Scan).